testsuites/CBLTester/upgrade_tests/test_cbl_upgrade.py

```diff
@@ -341,6 +341,8 @@ def _upgrade_db(args):
         prebuilt_db_path = "/assets/{}.cblite2.zip".format(old_liteserv_db_name)
     elif liteserv_platform == "xamarin-android":
         prebuilt_db_path = "{}.cblite2.zip".format(old_liteserv_db_name)
+    elif liteserv_platform in ["java-macosx", "java-msft", "java-ubuntu", "java-centos", "javaws-macosx", "javaws-msft", "javaws-ubuntu", "javaws-centos"]:
```

Comment:
same logic as 'xamarin-android'. can you combine or else you can rearrange and have this on else condition